hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/ContractResultServiceImpl.java

```diff
@@ -88,15 +89,17 @@ public void process(@NonNull RecordItem recordItem, Transaction transaction) {
 
         // contractResult
         var transactionHandler = transactionHandlerFactory.get(TransactionType.of(transaction.getType()));
-
+        var throwRecoverableError = shouldThrowRecoverableError(functionResult, recordItem, transactionRecord);
```

Comment:
When I said separate I meant to do a similar change for the receipt contractId.
```suggestion
        var throwRecoverableError = !(recordItem.isSuccessful() && transactionRecord.getReceipt().getContractID().hasEvmAddress());
```

Reply:
Got it, this has now been updated.